asm/main.s

```diff
@@ -41,11 +41,11 @@ _02000808:
 	ldr r1, _02000924 @ =0x07000000
 	mov r2, #1024
 	bl sub_200093C
-	ldr r1, _02000928 @ = 0x02000B68
+	ldr r1, _02000928 @ = _02000B68
```

Comment:
Labels should indicate what type they are or have a name matching the ghidra decomp.

Reply:
We're not going to have a clue what anything is

This is why early stuff in gen 3 dism is gUnknown_(Address) until identified

Reply:
At least that would show it's not e.g. a function or something like that.

Reply:
I don't think that's a problem for now